gnosis/protocol/gnosis_protocol_api.py

```diff
@@ -193,14 +206,36 @@ def get_estimated_amount(
         base_token: ChecksumAddress,
         quote_token: ChecksumAddress,
         kind: OrderKind,
-        amount: int,
+        amount_wei: int,
     ) -> Union[AmountResponse, ErrorResponse]:
         """
-        The estimated amount in quote token for either buying or selling amount of baseToken.
+
+        :param base_token:
+        :param quote_token:
+        :param kind:
+        :param amount_wei:
+        :return: Both `sellAmount` and `buyAmount` as they can be adjusted by CowSwap API
         """
-        url = self.base_url + f"markets/{base_token}-{quote_token}/{kind.name}/{amount}"
-        r = self.http_session.get(url)
-        if r.ok:
-            return AmountResponse(r.json())
+        order = Order(
+            sellToken=base_token,
+            buyToken=quote_token,
+            receiver=NULL_ADDRESS,
+            sellAmount=amount_wei * 10,
+            buyAmount=0,
```

Comment:
buyAmount=0 are you sure about this? here if the response from cowswap return 0 then the result is going to be 0 always. 
https://github.com/safe-global/safe-eth-py/pull/445/files#diff-800f25d9436120b4a629a7266b15f2b500b0a01ec5d7ca55c8509a6098f0be1cR63

Reply:
You are right, fixed